README.md

```diff
@@ -43,6 +43,37 @@ The services provided are:
 * `api` - convenience environment scoped to the `api` gem in the `/app/api` directory.
 * `sdk` - convenience environment scoped to the `sdk` gem in the `/app/sdk` directory.
 
+## Quick Start
+
+```ruby
+require 'opentelemetry/sdk'
+
+# Set preferred tracer implementation:
+SDK = OpenTelemetry::SDK
+
+factory = OpenTelemetry.tracer_factory = SDK::Trace::TracerFactory.new
```

Comment:
In the long haul, I think we are definitely in need of a `OpenTelemetry::SDK.configure` block to help users get everything setup (and possibly validate that they've setup everything up properly). For now, this is the state of the world.

Reply:
Agreed - this hurts every time I read it 😞